bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/ExternalProcessExecutor.java

```diff
@@ -79,12 +77,35 @@ public boolean exec(List<String> cmd, Duration timeout) {
 			return false;
 		}
 
+		// Capture error output
+		StringBuilder errorOutput = new StringBuilder();
+		Thread errorReader = new Thread(() -> {

```

Comment:
Since we already use `nuprocess` in the project, we could replace this manual Thread-based error stream handling with it. `nuprocess` handles I/O more efficiently without needing separate threads, reducing complexity and avoiding potential concurrency issues.

Reply:
Thanks for the suggestion. Now it's using `nuprocess`.